photo-app/app/models/user.rb

```diff
@@ -3,4 +3,6 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable, :confirmable,
          :recoverable, :rememberable, :validatable, :trackable
+  has_one :payment
```

Comment:
huh? 1 payment record only?
The user cannot make many payments?

Reply:
Ya cuz they only need to make the payment once when they sign up to the site

Reply:
what if later on if the business side decide that they want to change it into a subscription?
Or, based on the current logic, what if a payment fail?